index.html

```diff
@@ -168,6 +168,21 @@ <h3 class="language-title">Skills</h3>
                 </div>
             </div>
         </section>
+        <section class="section-contuct-me">
+            <div class="container">
+                <div class="contuct-me-box">
+                    <h2 class="contuct-me__heading">Contact me</h2>
+                    <p class="contuct-me__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+                    
+                    <form action="#" method="post">
```

Comment:
- [ ] Your form design is good and very similar to the template, but your form is not connected to the Formspree service and its not working. Kindly follow the 'Getting Started' section of the [Formspree service](https://formspree.io/html), and add the action with your id so it can work properly and users can send you messages.